### PR TITLE
OA-10:Add a new model class for mapping launch value of SMART apps

### PR DESCRIPTION
**Pull request by PKatGITHUB**
My work on [OA-10:Add a new model class for mapping launch value of SMART apps](https://issues.openmrs.org/browse/OA-10)

@mavrk Please have a look! :)

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/oauth2/api/model/LaunchValue.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
+package org.openmrs.module.oauth2.api.model;
```
Comment (mavrk) [inline api/src/main/java/org/openmrs/module/oauth2/api/model/LaunchValue.java]: Let's move the SMART models to oauth2.api.mode.smart ? And I mean both the models (OA-9 and OA-10). Make this change and update the PR with a combined squashed commit (OA-9 and OA-10) 😄

**Comment by mavrk** [comment]
Also, copy the file for OA-9 and git pull --rebase upstream master

File: api/src/main/java/org/openmrs/module/oauth2/api/model/LaunchValue.java (hunk 1 above)
Comment (PKatGITHUB) [reply api/src/main/java/org/openmrs/module/oauth2/api/model/LaunchValue.java]: OK 👍

**Comment by PKatGITHUB** [comment]
shouldn't it be git pull --rebase upstream gsoc-18? as we are working on gsoc-18 branch :)

**Comment by mavrk** [comment]
Yes 😝

**Comment by PKatGITHUB** [comment]
Done! 👍

**Pull request merged**